sympy/printing/tests/test_repr.py

```diff
@@ -101,12 +101,11 @@ def test_Rational():
 
 
 def test_Float():
-    sT(Float('1.23', prec=3), "Float('1.22998', prec=3)")
```

Comment:
You said it yourself:

> to make sure that no precision in lost between translations from decimal to binary

(This means between decimal and binary *representations* of the number, not between `dps` and `precision`.)

> not for any further computation.

No so, `eval(srepr(x))` should give exact same object whenever possible.